src/SprykerSdk/AsyncApi/Message/AsyncApiError.php

```diff
@@ -19,6 +19,16 @@ class AsyncApiError
      */
     protected const CODE_GENERATION_ERROR_PREFIX = 'AsyncAPI code generation error';
 
+    protected static ?bool $isNotWindows = null;
+
+    /**
+     * @param bool|null $isNotWindows
+     */
+    public function __construct(?bool $isNotWindows = null)
+    {
+        static::$isNotWindows = $isNotWindows ?? (PHP_SAPI === PHP_SAPI && stripos(PHP_OS, 'WIN') === false);
```

Comment:
The question to answer is is this constant used more than once and would a descriptive constant name be better than the plain value?

If no, do not overuse constants as they should support reading the code and not because of we can have one.